### ISSUE TITLE
update about me

### ISSUE THREAD

**Issue by devloos** [devloos/Portifolio#4]

**Comment by devloos** [comment]
I'm an enthusiastic software engineer with a deep passion for Computer Science. My journey into this field started in 2020, but it wasn't until November 2021 that I could actively pursue it due to conflicting priorities. Once I began, I quickly fell in love with various Computer Science concepts. I focused on understanding C++ through online tutorials and built projects like the [Snake Game,](https://github.com/devloos/Snake-Game) a [Pokemon Clone,](https://github.com/devloos/Pokemon-Clone) and a [Sales System](https://github.com/devloos/Sales-System) using libraries like ncurses, RayLib, and mongocxx.

Later, I enrolled in college and have been excelling, maintaining an impressive 3.85 GPA while pursuing Computer Science. My goal is to get accepted into the University of California for fall semester of 2024, a significant achievement as I'd be the first in my family to attend a university.

In 2021, while working as a supervisor at Ethika, my passion for computers grew, and I started demonstrating my knowledge. This led to my first software development job, marking a pivotal moment in my career.

I'm dedicated to continuous learning and adoption of many new tools. Alongside my passion for the field, I enjoy fishing, playing soccer, and spending time with my family. A life goal of mine is to create something that leaves a meaningful impact, regardless of its scale.

**Comment by devloos** [comment]
I'm an enthusiastic software engineer with a deep passion for Computer Science. My journey into this field started in 2020, but it wasn't until November 2021 that I could actively pursue it due to conflicting priorities. Once I began, I quickly fell in love with various Computer Science concepts. I focused on understanding C++ through online tutorials and built projects like the [Snake Game,](https://github.com/devloos/Snake-Game) a [Pokemon Clone,](https://github.com/devloos/Pokemon-Clone) and a [Sales System](https://github.com/devloos/Sales-System) using libraries like ncurses, RayLib, and mongocxx.

Later, I enrolled in Long Beach State University and have been excelling, maintaining an impressive 3.85 GPA while pursuing Computer Science. A significant achievement as I'd be the first in my family to attend a university.

Around that same time, while working as a supervisor at Ethika, my passion for computers grew, and I started demonstrating my knowledge. This led to my first software development job, marking a pivotal moment in my career.

I'm dedicated to continuous learning and the adoption of many new tools. Alongside my passion for the field, I enjoy fishing, playing soccer, and spending time with my family. A life goal of mine is to create something that leaves a meaningful impact, regardless of its scale.
